src/utilities/FormManager.ts

```diff
@@ -84,15 +85,21 @@ class FormManager {
     return this.formWrappedInstance.props.form;
   }
 
-  public get fieldConfigs (): IFieldConfig[] {
-    return getFieldSetsFields(this.args.fieldSets);
-  }
-
   public get isFormDisabled (): boolean {
     // The disabled prop can be changed any time, so we can't just save it locally
     return this.isSaving || !!this.formWrappedInstance.props.disabled;
   }
 
+  public get fieldSets (): IFieldSet[] {
+    // The fieldSets prop can be changed any time, so try to get them dynamically if you can
+    const fieldSetsProp = this.formWrappedInstance.props.fieldSets;
+    return fieldSetsProp ? fillInFieldSets(fieldSetsProp) : this.args.fieldSets;
```

Comment:
Is there any case when we would want to defer to the passed field sets instead of the form props? I can't think of one, but just want to make sure

Reply:
No, I don't think so. The reason we're doing this is in the off chance that `formWrappedInstance` doesn't take a `fieldSets` prop, like if someone instantiates it themselves somewhere.